### DISCUSSION TITLE
Upgradeing to 1.25.0-1 - Q about Breaking Changes

### DISCUSSION THREAD

**Discussion by smartroad** [Koenkk/zigbee2mqtt#12141] [General]
Hi!

I have a Home Assistant install and it is saying Zigbee2mqtt is upgrading to 1.25 (from 1.24). I am confused by the Breaking Changes message. 

It says about it won't start with an MQTT Server or serial adapter and muse be configured though the add-on config page, but then in the next line it says you can remove the config from the config page.

As I have a working 1.24 install, what will happen if I upgrade? Is there anything I need to do? I have Mosquitto broker installed, is that what it means about the MQTT Server?

Sorry for being so stupid!

**Comment by Stooovie** [comment]
I can only state my experience:
- upgrading to Z2M 1.25.0 was fine, HA is fully updated to 2022.4.5
- after seeing warnings about the old addon config, I proceeded to reset it (saving the old one to an external file first)
- after restarting Z2M, it was empty, zero devices
- restoring the old config, all devices are back, they do work in Z2M BUT none of the commands/state changes make it to HA, at all

So now I'm left with half of my devices not working and unsure what to do.

**Reply by smartroad** [reply]
Thanks for letting me know your experience. For now I am staying at 1.24 as it is working and I don't have the time or energy to mess about getting it working again if it were to break.

Did you get anywhere with your setup after restoring?
